### ISSUE TITLE
Passwords for ProxyJump hosts in .ssh/config are not saved 

### ISSUE THREAD

**Issue by dirkpetersen** [iterate-ch/cyberduck#13936]
**Is your feature request related to a problem? Please describe.**

When I use the ProxyJump feature the password for the jumphost is never saved (Version 8.5.1 / Windows 11). This is especially problematic since Cyberduck seems to need to re-authenticate each time I hit the upload button so if the files I upload are scattered over multiple folders I have to enter my password 4-5 times which seems a bit hacky

```
Host jumper
    User user1
    Hostname host1.domain.org

Host *
    ProxyJump jumper
```

**Describe the solution you'd like**
save password for jumphost if checked when prompted 

**Describe alternatives you've considered**
Considered ssh keys but are not allowed here

**Comment by dkocher** [comment]
Should have been resolved with 3e28ad3b but apparently we fail to lookup the saved credentials.

**Comment by dkocher** [comment]
> Awesome, thanks a lot !

It would be great if you can verify the fix with build 8.5.2.38764 available through _Preferences… → Update → Snapshot Builds_.

**Comment by dirkpetersen** [comment]
I tested it and it works very well. Thank you!
